src/app/components/BibPage/BibPage.jsx

```diff
@@ -29,7 +31,7 @@ const BibPage = (props) => {
     shortenItems = false;
   }
 
-  const itemHoldings = items.length ?
+  const itemHoldings = items.length && !electronicItems ?
```

Comment:
Just in case all the items are electronic resources, then we don't want to display the whole table at all.